### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_AI4D:The edit box present in row with controller under "storage settings" is not set.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9294]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#AI4D;#MAS4.2.1;#MAS4.1.2;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tool:Accessibility insights for desktop
MAS References:MAS4.1.2, MAS4.2.1, MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.
8. Launch accessibility insights for desktop .

**Actual:**
Issue1:
On hovering over the edit fields under the "Storage settings" button the names for them are not set.
All 3 edit after the first one has no name.

Issue2:
The name of the first one does not match its label.

**Expected:**
All the edit fields should have a name set for them as the labels above it "Storage class for data, Claim size for data, storage class for logs, claim size for logs"

**User Impact:**
Screen reader users will not know what is the purpose of the field, they won't know what they need to enter the field.

**Attachment for Reference :**
![image](https://user-images.githubusercontent.com/24580570/75137258-2c70c200-570d-11ea-9cfd-9ccd8ccc8a25.png)

**Comment by alanrenmsft** [comment]
dupe of #9345

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags:** #Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
